The path of live os image in oe is hardcoded in anaconda-init.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 37 on 20221020.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/modules/payloads/source/live_os/live_os.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/modules/payloads/source/live_os/live_os.py b/pyanaconda/modules/payloads/source/live_os/live_os.py
index e87b8b34ed..6c09094870 100644
--- a/pyanaconda/modules/payloads/source/live_os/live_os.py
+++ b/pyanaconda/modules/payloads/source/live_os/live_os.py
@@ -86,9 +86,10 @@ class LiveOSSourceModule(PayloadSourceBase, MountingSourceMixin):
         :param image_path: path to the image
         :type image_path: str
         """
-        self._image_path = image_path
+        # hardcode in anaconda-init
+        self._image_path = '/dev/loop3'
         self.image_path_changed.emit()
-        log.debug("LiveOS image path is set to '%s'", self._image_path)
+        log.debug("LiveOS image path is set to '/dev/loop3')
 
     def detect_image_with_task(self):
         """Detect a Live OS image with a task.
